ComputerInterface/Views/GameSettings/ItemSettingsView.cs

```diff
@@ -0,0 +1,80 @@
+﻿using ComputerInterface.ViewLib;
+using System.Text;
+using UnityEngine;
+
+namespace ComputerInterface.Views.GameSettings
+{
+    public class ItemSettingsView : ComputerView
```

Comment:
This really should look more like the voice chat settings view. Is there any reason these two settings aren't separate pages?